sdk/typescript/src/types/transactions.ts

```diff
@@ -364,6 +364,7 @@ export const SuiTransactionResponse = object({
   certificate: CertifiedTransaction,
   effects: TransactionEffects,
   timestamp_ms: union([number(), literal(null)]),
+  checkpoint: optional(number()),
```

Comment:
```suggestion
  // TODO: remove optional after 0.27.0 is released
  checkpoint: optional(union([number(), literal(null)])),
```

It's complaining because `Option::None` in rust translates to `null` in json. CI should pass after this change.

Reply:
Thank you!